command/s3_get.go

```diff
@@ -195,30 +212,10 @@ func (c *s3get) getWithRetry(ctx context.Context, logger client.LoggerProducer)
 
 // Fetch the specified resource from s3.
 func (c *s3get) get(ctx context.Context) error {
-	// get the appropriate session and bucket
-	auth := &aws.Auth{
-		AccessKey: c.AwsKey,
-		SecretKey: c.AwsSecret,
-	}
-
-	client := util.GetHTTPClient()
-	defer util.PutHTTPClient(client)
```

Comment:
how does pail get an http client? we should make sure that we don't leak connections by switching to pail.